tests/conftest.py

```diff
@@ -1,6 +1,7 @@
 import datetime
-from fauxfactory import gen_string
 import pytest
+import yaml
```

Comment:
This will need pyyaml module. Do you want to add pyyaml dependency ?

Reply:
Ignore this, it comes from ansible 
`Collecting PyYAML (from ansible==2.6.17->-r requirements.txt (line 1))`